gradle-scalatest.plugin

```diff
@@ -0,0 +1,21 @@
+buildscript {
+	repositories {
+		maven {
+			url "https://plugins.gradle.org/m2/"
+		}
+	}
+	dependencies {
+		classpath "com.github.maiflai:gradle-scalatest:0.8"
+	}
+}
+
+if (!project.plugins.findPlugin(com.github.maiflai.ScalaTestPlugin)) {
```

Comment:
I'm not sure why this is necessary, .apply plugin` has been idempotent in my experience. The way that this code is being called is from a `subprojects {}` which means the apply call will work on each new (sub)project evaluated.

AFAICT this should be safe to nuke